frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinition.tsx

```diff
@@ -136,7 +143,13 @@ const NADListEmpty: React.FC = () => {
   return (
     <EmptyState>
       <EmptyStateHeader
-        titleText={<>{t('kubevirt-plugin~No network attachment definitions found')}</>}
+        titleText={
+          <>
+            {t('kubevirt-plugin~No {{label}} found', {
```

Comment:
Shouldn't be plural used here instead? Originally, we use plural when showing something not found, in the list views.

Reply:
yes you're right